tests/unit/setup-context-test.js

```diff
@@ -0,0 +1,110 @@
+import { module, test } from 'qunit';
+import Service from '@ember/service';
+import { setupContext } from 'ember-test-helpers';
+import hasEmberVersion from 'ember-test-helpers/has-ember-version';
+import { setResolverRegistry, createCustomResolver } from '../helpers/resolver';
+
+module('setupContext', function(hooks) {
+  hooks.before(function() {
+    setResolverRegistry({
+      'service:foo': Service.extend({ isFoo: true }),
+    });
+  });
+
+  module('without options', function(hooks) {
+    let context;
+    hooks.beforeEach(function() {
+      context = {};
+      setupContext(context);
+    });
+
+    test('it sets up this.owner', function(assert) {
+      let { owner } = context;
+      assert.ok(owner, 'owner was setup');
+      assert.equal(
+        typeof owner.lookup,
+        'function',
+        'has expected lookup interface'
+      );
+
+      if (hasEmberVersion(2, 12)) {
+        assert.equal(
+          typeof owner.factoryFor,
+          'function',
+          'has expected lookup interface'
+        );
+      }
+    });
+
+    test('it uses the default resolver if no override specified', function(
+      assert
+    ) {
+      let { owner } = context;
+      let instance = owner.lookup('service:foo');
+      assert.equal(instance.isFoo, true, 'uses the default resolver');
```

Comment:
wouldn't the `lookup()` call already fail and cause an exception?

Reply:
No, lookup will return `undefined` if it does not find a result.

Reply:
> No, owner.lookup will return undefined if it does not find a factory to instantiate.

exactly, and then evaluating `instance.isFoo` would result in an `undefined has no isFoo property` error instead of a failing assertion.

Reply:
Sorry, I don't grok what you mean.  Are you saying that I should remove the `assert.equal(instance.isFoo, true` line? Or something else?

Reply:
I guess what I'm saying is

```
assert.ok(owner.lookup('service:foo'))
```

would probably be sufficient here and result in a proper failing assertion instead of throwing an Error

Reply:
Ah, I see.  Ya, I wanted to make sure that the "default" resolver was used (which has `isFoo` set). There is a test down below that also has a `service:foo` but not `isFoo: true`.